src/oaklib/cli.py

```diff
@@ -1032,13 +1032,20 @@ def term_metadata(terms, reification: bool, output_type: str, output: str):
     "-L",
     help="path to lexical index. This is recreated each time unless --no-recreate is passed",
 )
+@click.option(
+    "--exclude-terms",
+    "-x",
+    multiple=True,
+    help="Text file or list of terms to exclude from annotation. Each newline separated entry is a distinct text.",
```

Comment:
I suggest `--exclude-tokens` and for docs "file or list of tokens to filter from input prior to annotation. If passed as text file, each newline separated entry is a distinct token"

Reply:
Done